### DISCUSSION TITLE
Inverse Pole Figure Change default Corner orientations

### DISCUSSION THREAD

**Discussion by WenxiLee** [mtex-toolbox/mtex#1329] [Ask Anything]
Hi all,

I want to change my IPF figure to another triangle(I mean, [001,011,1-11] to [001,101,111]). Could you give me some suggestions? Thank you!
![image](https://user-images.githubusercontent.com/98835657/152010959-abdb48e9-5161-4955-8495-b2b1c3c1c51f.png)
![image](https://user-images.githubusercontent.com/98835657/152011680-79905577-d7a5-4053-9953-fffc2452297e.png)

**Comment by kilir** [comment]
Hi,
what is the exact reason you'd like to do that? The directions are equivalent.
E.g.:
```matlab
cs = crystalSymmetry('cubic')
h= Miller({0 0 1},{1 0 1},{1 1 1},cs)

o = orientation.rand(5,cs)

f= sphericalRegion.byVertices(h)
plot(f)
hold on
plotIPDF(o,xvector)
hold off
annotate(h.symmetrise('unique'),'labeled')


% default
nextAxis
plotIPDF(o,xvector)

```
Cheers,
Rüdiger

**Reply by WenxiLee** [reply]
Hi Rüdiger,

Thank you! That works for me! Yeah, I know it's the same due to the symmetry. I just want to change to a more common orientation triangle and do some orientation calculations. Again, thanks for your help! 

Best,
Wenxi

**Comment by qmrphd** [comment]
is there any to put indentation hardness or modulus on IPF figure.
